en/releases/centreon-commercial-extensions.md

```diff
@@ -271,6 +271,19 @@ If you have feature requests or want to report a bug, please contact support.
 
 ## Centreon MBI
 
+## 20.04.7
+
+`July 15, 2021`
+
+#### Improvements
+
+- Optimized widget requests
+
+#### Bugfixes
+
+- Removed old credentials from the repo
```

Comment:
This is just creds used for tests, it has no impact on the product itself.
Not sure we should put that in the RN.
```suggestion
```